### PR TITLE
Add first consumer/provider contract tests for getUserById

### PR DESCRIPTION
**Pull request by akeshmiri**

### CODE REVIEW

File: athena-api-boot/src/main/java/org/catools/athena/rest/core/controller/AthenaEnvironmentController.java [hunk 1]
```diff
@@ -42,47 +41,65 @@ public AthenaEnvironmentController(AthenaCoreService athenaCoreService) {
     public ResponseEntity<Set<EnvironmentDto>> getEnvironments() {
         final Set<EnvironmentDto> environments = athenaCoreService.getEnvironments();
         return environments.isEmpty() ?
-                ResponseEntity.noContent().cacheControl(CACHE_MAX_AGE_ONE_HOUR).build() :
-                ResponseEntity.ok().cacheControl(CACHE_MAX_AGE_ONE_HOUR).body(environments);
+                ResponseEntityUtils.noContent() :
```
Comment (natalaly) [inline athena-api-boot/src/main/java/org/catools/athena/rest/core/controller/AthenaEnvironmentController.java:44]: Nice optimization

File: athena-api-boot/src/main/java/org/catools/athena/rest/core/controller/AthenaUserController.java [hunk 2]
```diff
@@ -41,46 +40,62 @@ public AthenaUserController(AthenaCoreService athenaCoreService) {
...
+                ResponseEntityUtils.ok(users);
     }
 
-    @GetMapping(USER)
+    @GetMapping(USER_PATH)
     @Operation(
             summary = "Retrieve user by name",
             responses = {
                     @ApiResponse(responseCode = "200", description = "Successfully returned data"),
                     @ApiResponse(responseCode = "204", description = "No content to return")
```
Comment (natalaly) [inline athena-api-boot/src/main/java/org/catools/athena/rest/core/controller/AthenaUserController.java:52]: I would add 400 for the case if username is not provided

Comment (natalaly) [reply athena-api-boot/src/main/java/org/catools/athena/rest/core/controller/AthenaUserController.java:52]: Also I believe that we should handle cases when username is not provided. I can do it in a separate PR if you want me to work on it.

Comment (natalaly) [reply athena-api-boot/src/main/java/org/catools/athena/rest/core/controller/AthenaUserController.java:52]: Same for other parameters we send to methods get methods in services and controllers. Probably we need to talk about where we want to handle this.

**Review by natalaly — APPROVED** [review]
Considering that we have some exception handling to do, I recommend to do it in a separate PR. if agree, feel free to merge and I will work on Exception handling

File: athena-api-boot/src/main/java/org/catools/athena/rest/core/controller/AthenaUserController.java (hunk 2 above)
Comment (akeshmiri) [reply athena-api-boot/src/main/java/org/catools/athena/rest/core/controller/AthenaUserController.java:52]: Agree, lets do it in a separate PR, and sure, appreciate if you take care of it

**Pull request merged**